<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>400</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::CustomContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	background-color: rgb(215,215,215);
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_deposit">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>350</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #FFC0CB;
	color: black;
	border: 1px solid gray;
	font-size: 16px;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #DB7093, stop: 1 #FFC0CB);
}</string>
    </property>
    <property name="text">
     <string>Deposit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_credit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>350</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #FFC0CB;
	color: black;
	border: 1px solid gray;
	font-size: 16px;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #DB7093, stop: 1 #FFC0CB);
}</string>
    </property>
    <property name="text">
     <string>Credit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_close_bracket">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>150</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #FFC0CB;
	color: black;
	border: 1px solid gray;
	font-size: 16px;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #DB7093, stop: 1 #FFC0CB);
}</string>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_zero">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>350</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(255, 255, 255);
	color: black;
	border: 1px solid gray;
	font-size: 16px;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_one">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>300</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(255, 255, 255);
	color: black;
	border: 1px solid gray;
	font-size: 16px;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_two">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>300</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(255, 255, 255);
	color: black;
	border: 1px solid gray;
	font-size: 16px;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_three">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>300</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(255, 255, 255);
	color: black;
	border: 1px solid gray;
	font-size: 16px;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_four">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>250</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(255, 255, 255);
	color: black;
	border: 1px solid gray;
	font-size: 16px;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_five">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(255, 255, 255);
	color: black;
	border: 1px solid gray;
	font-size: 16px;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_six">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>250</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(255, 255, 255);
	color: black;
	border: 1px solid gray;
	font-size: 16px;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_seven">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(255, 255, 255);
	color: black;
	border: 1px solid gray;
	font-size: 16px;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_eigth">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(255, 255, 255);
	color: black;
	border: 1px solid gray;
	font-size: 16px;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_nine">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>200</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(255, 255, 255);
	color: black;
	border: 1px solid gray;
	font-size: 16px;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_point">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>350</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(190, 190, 190);
	color: black;
	border: 1px solid gray;
	font-size: 16px;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0,  y2: 1, stop: 0 #A9A9A9, stop: 1 #A9A9A9);
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_calculate">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>350</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #00FFFF;
	color: black;
	border: 1px solid gray;
	font-size: 16px;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #00CED1, stop: 1 #00CDE1);
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_add">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>300</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #00FFFF;
	color: black;
	border: 1px solid gray;
	font-size: 16px;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #00CED1, stop: 1 #00CDE1);
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sub">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>250</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #00FFFF;
	color: black;
	border: 1px solid gray;
	font-size: 16px;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #00CED1, stop: 1 #00CDE1);
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mul">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>200</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #00FFFF;
	color: black;
	border: 1px solid gray;
	font-size: 16px;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #00CED1, stop: 1 #00CDE1);
}</string>
    </property>
    <property name="text">
     <string>×</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_div">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #00FFFF;
	color: black;
	border: 1px solid gray;
	font-size: 16px;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #00CED1, stop: 1 #00CDE1);
}</string>
    </property>
    <property name="text">
     <string>÷</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_delete_one">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(190, 190, 190);
	color: black;
	border: 1px solid gray;
	font-size: 16px;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0,  y2: 1, stop: 0 #A9A9A9, stop: 1 #A9A9A9);
}</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_delete_all">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(190, 190, 190);
	color: black;
	border: 1px solid gray;
	font-size: 16px;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0,  y2: 1, stop: 0 #A9A9A9, stop: 1 #A9A9A9);
}</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_percent">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>150</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(190, 190, 190);
	color: black;
	border: 1px solid gray;
	font-size: 16px;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0,  y2: 1, stop: 0 #A9A9A9, stop: 1 #A9A9A9);
}</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sin">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>150</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #00FF7F;
	color: black;
	border: 1px solid gray;
	font-size: 16px;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #00cd7f, stop: 1 #00e17f);
}</string>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cos">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>150</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #00FF7F;
	color: black;
	border: 1px solid gray;
	font-size: 16px;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #00cd7f, stop: 1 #00e17f);
}</string>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_tan">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>150</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #00FF7F;
	color: black;
	border: 1px solid gray;
	font-size: 16px;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #00cd7f, stop: 1 #00e17f);
}</string>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_asin">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>200</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #00FF7F;
	color: black;
	border: 1px solid gray;
	font-size: 16px;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #00cd7f, stop: 1 #00e17f);
}</string>
    </property>
    <property name="text">
     <string>asin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_acos">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>200</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #00FF7F;
	color: black;
	border: 1px solid gray;
	font-size: 16px;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #00cd7f, stop: 1 #00e17f);
}</string>
    </property>
    <property name="text">
     <string>acos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_atan">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>200</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #00FF7F;
	color: black;
	border: 1px solid gray;
	font-size: 16px;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #00cd7f, stop: 1 #00e17f);
}</string>
    </property>
    <property name="text">
     <string>atan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sqrt">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>250</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #00FF7F;
	color: black;
	border: 1px solid gray;
	font-size: 16px;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #00cd7f, stop: 1 #00e17f);
}</string>
    </property>
    <property name="text">
     <string>sqrt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ln">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>250</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #00FF7F;
	color: black;
	border: 1px solid gray;
	font-size: 16px;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #00cd7f, stop: 1 #00e17f);
}</string>
    </property>
    <property name="text">
     <string>ln</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_log">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>250</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #00FF7F;
	color: black;
	border: 1px solid gray;
	font-size: 16px;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #00cd7f, stop: 1 #00e17f);
}</string>
    </property>
    <property name="text">
     <string>log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mod">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>300</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #00FF7F;
	color: black;
	border: 1px solid gray;
	font-size: 16px;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #00cd7f, stop: 1 #00e17f);
}</string>
    </property>
    <property name="text">
     <string>mod</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_X">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>300</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #00FF7F;
	color: black;
	border: 1px solid gray;
	font-size: 16px;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #00cd7f, stop: 1 #00e17f);
}</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pow">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>300</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #00FF7F;
	color: black;
	border: 1px solid gray;
	font-size: 16px;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #00cd7f, stop: 1 #00e17f);
}</string>
    </property>
    <property name="text">
     <string>x^</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_result">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>400</width>
      <height>150</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	background-color: rgb(215,215,215);
	color: black;
	border: 1px solid gray;
	font-size: 24px
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_graph">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>250</y>
      <width>50</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #FFC0CB;
	color: black;
	border: 1px solid gray;
	font-size: 16px;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #DB7093, stop: 1 #FFC0CB);
}</string>
    </property>
    <property name="text">
     <string>Graph</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>0</y>
      <width>500</width>
      <height>400</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_x_max">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>50</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	background-color: rgb(215,215,215);
	color: black;
	border: 1px solid gray;
	font-size: 24px
}</string>
    </property>
    <property name="text">
     <string>X  max</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_x_max_2">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>150</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	background-color: rgb(215,215,215);
	color: black;
	border: 1px solid gray;
	font-size: 24px
}</string>
    </property>
    <property name="text">
     <string>X  min</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_make_graph">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>350</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color:  rgb(215,215,215);
	color: black;
	border: 1px solid gray;
	font-size: 16px;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0,  y2: 1, stop: 0 #A9A9A9, stop: 1 #A9A9A9);
}</string>
    </property>
    <property name="text">
     <string>Make Graph</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_x_max">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>80</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	color: black;
	background-color: rgb(255,255,255);
}</string>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_x_min">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>180</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	color: black;
	background-color: rgb(255,255,255);
}</string>
    </property>
    <property name="text">
     <string>-10</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_open_bracket">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>200</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #FFC0CB;
	color: black;
	border: 1px solid gray;
	font-size: 16px;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #DB7093, stop: 1 #FFC0CB);
}</string>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_make_visual">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>250</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color:  rgb(215,215,215);
	color: black;
	border: 1px solid gray;
	font-size: 16px;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0,  y2: 1, stop: 0 #A9A9A9, stop: 1 #A9A9A9);
}</string>
    </property>
    <property name="text">
     <string>Make Visual</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
